apps/ledger-live-desktop/src/newArch/features/Collectibles/components/Collection/TableRow/index.tsx

```diff
@@ -25,6 +25,13 @@ const Container = styled(Box)`
   }
 `;
 
+const SatsIconContainer = styled(Flex)`
+  border-radius: 8px;
+  background: ${p => p.theme.colors.opacityDefault.c05};
+  border: 1px solid ${p => p.theme.colors.opacityDefault.c10};
+  padding: 8px;
+`;
+
```

Comment:
You maybe should put this outside of TableRow as it's specific to Stats ?

Reply:
Yes I've changed that in [feat/LIVE-12479_b](https://github.com/LedgerHQ/ledger-live/pull/7756). There will be a dedicated component